services/web/server/src/simcore_service_webserver/projects/_projects_service.py

```diff
@@ -187,19 +190,26 @@
 _logger = logging.getLogger(__name__)
 
 
-async def conditionally_unsubscribe_from_project_logs(
+async def conditionally_unsubscribe_project_logs_across_replicas(
     app: web.Application, project_id: ProjectID, user_id: UserID
 ) -> None:
     """
     Unsubscribes from project logs only if no active socket connections remain for the project.
```

Comment:
add the meaning of 'user_id` in the comment i.e. a given user_id unsubscribes from the project logs